### ISSUE TITLE
Cloudwatch Log Event Subscription

### ISSUE THREAD

**Issue by lukas-gitl** [serverless/serverless#1286]
Would very much like the ability to subscribe a lambda function to Cloudwatch Logs.

Currently only dynamodbstream, kinesisstream, s3, sns, and schedule are supported according to the docs.

**Comment by homingli** [comment]
I had a PR for this #1070 but I was told there will be "a very different way of setting up events"

**Comment by flomotlik** [comment]
@lukas-gitl @homingli through our new Cloudformation config in V1 you should be able to easily set this up through custom resources

**Comment by pas256** [comment]
Where is the documentation for setting up a custom resource like this? I couldn't find it.

**Comment by andymac4182** [comment]
@lukas-gitl I did this with a custom plugin since I wanted to subscribe 1 lambda function to the cloudwatch logs of all the other functions in the service. 
@pas256 I am happy to write some documentation similar to the DynamoDb one if that would help?

**Comment by pas256** [comment]
Yes it would, thanks.

**Comment by andymac4182** [comment]
@pas256 Here is my plugin. https://gist.github.com/andymac4182/4837f722231ea493685f6b4699c939a1 I will see if I have time today to write some documentation around it. Hopefully this helps.

**Comment by flomotlik** [comment]
@pas256 in general here is the documentation for custom provider resources: https://github.com/serverless/serverless/blob/master/docs/01-guide/06-custom-provider-resources.md

Its just Cloudformation syntax, so you can simply reference any of the created functions and set up your custom events.

**Comment by love8587** [comment]
@andymac4182 Hello, I'm happy to find your solution. 
I want to integrate between Cloud Watch Log Stream <-> Lambda function.
It's very simple and easy in AWS Console.
But in the Severless, It's not supported yet. So I'm trying another solution so much..

I almost give up it. But when I found out your git source, It seem to work for me. 

How Can I use it? (Sorry, I've just used Serverless for 3 days)
